frontend/src/components/IstioStatus/IstioStatus.tsx

```diff
@@ -19,12 +19,16 @@ import { KialiDispatch } from 'types/Redux';
 import { NamespaceThunkActions } from '../../actions/NamespaceThunkActions';
 import { connectRefresh } from '../Refresh/connectRefresh';
 import { kialiStyle } from 'styles/StyleUtils';
+import { IconProps, createIcon } from 'config/KialiIcon';
 
-type ReduxProps = {
+type ReduxStateProps = {
```

Comment:
Not sure splitting ReduxProps is necessary but it's fine, it's still clear that these are connected props.